lib/src/pdfjs_global.dart

```diff
@@ -29,113 +29,159 @@ enum VERBOSITY_LEVELS {
 }
 
 class PDFJS {
-  static final JsObject _pdfjsContext = context['pdfjsLib'] as JsObject;
+  /// the local copy of versionSafePdfJsContext once initialized
+  static JsObject _pdfJsContext;
+
+  /// the local copy of versionSafePdfJsViewerContext once initialized
+  static JsObject _pdfJsViewerContext;
+
+  /// Getter for obtaining the pdfjs context for various versions
+  /// of pdf.js
+  static JsObject get versionSafePdfJsContext {
+    if(_pdfJsContext != null) {
+      return _pdfJsContext;
+    }
+    
+    // ^v2.5.207
+    if(context['pdfjsLib'] != null) {
```

Comment:
This is needed, as can be seen [here](https://github.com/Workiva/pdfjs_dart/pull/34/files?file-filters%5B%5D=.dart#diff-37e810e81493e61ea4411c448605a974e2b9009e500bd75c131914a845cbf90bR32).

v1.10.90.4 of the interop looks for `context['PDFJS']`, whereas newer versions look for `context['pdfjsLib']`